### PR TITLE
account.chpass: Show error when no accounts found

### PR DESCRIPTION
**Pull request by matthew-mcallister**
EDIT: Screenshot (updated):

screenshot_2017-09-15_17-41-20

### CODE REVIEW

**Comment by tahabi** [comment]
out of curiosity, how did you test this, by removing the calnetuid from your own ldap entry?

**Comment by matthew-mcallister** [comment]
I added the line `accounts = []` to the view. If the situation were more complex, I think I would try patching the ocflib functions with mock.

File: ocfweb/account/templates/account/partials/chpass-no-accounts.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<p>Didn't find any accounts for CalNet UID {{calnet_uid}}.</p>
+<p>
+    If you have an individual account and it isn't showing up here,
+    please <a href="{% url 'doc' 'contact' %}">contact us</a>.
```
Comment (chriskuehl) [inline ocfweb/account/templates/account/partials/chpass-no-accounts.html]: maybe ask them to include the calnet UID in the contact email? we ask that on other pages and it's usually pretty helpful to link accounts

Comment (matthew-mcallister) [reply ocfweb/account/templates/account/partials/chpass-no-accounts.html]: Oh, thanks, you're right, it's square on the same page.

**Pull request merged**